### PR TITLE
Always show options menu items. Resolves GH-2154

### PR DESCRIPTION
**Pull request by thisandagain**
Pending confirmation from @xmatthewx or @HPaulJohnson

### CODE REVIEW

**Comment by thisandagain** [comment]
Confirmed by @HPaulJohnson

**Comment by thisandagain** [comment]
@k88hudson Can you do an R on the code and make sure I didn't break anything on newer devices?

**Comment by k88hudson** [comment]
R+! Good thing we have people testing on Samsung devices, heh

File: app/src/main/java/org/mozilla/webmaker/WebmakerActivity.java [hunk 1]
```diff
@@ -52,6 +55,17 @@ protected void onCreate(Bundle savedInstanceState) {
             actionBar.setIcon(android.R.color.transparent);
         }
 
+        try {
+            ViewConfiguration config = ViewConfiguration.get(this);
+            Field menuKeyField = ViewConfiguration.class.getDeclaredField("sHasPermanentMenuKey");
```
Comment (k88hudson) [inline app/src/main/java/org/mozilla/webmaker/WebmakerActivity.java]: oh yeah totally obvious :stuck_out_tongue_winking_eye:

**Pull request merged**